microprofile.jdt/com.redhat.microprofile.jdt.core/plugin.xml

```diff
@@ -21,12 +22,25 @@
             <command id="microprofile/java/hover"/>
        </delegateCommandHandler>
    </extension>
+  
+   <!-- Delegate command handler for Java project types -->
+   <extension point="org.eclipse.jdt.ls.core.delegateCommandHandler">
+      <delegateCommandHandler class="com.redhat.microprofile.jdt.internal.core.ls.ProjectTypesDelegateCommandHandler">
```

Comment:
Rename to `JavaProjectDelegateCommandHandler` which is commnds about java project (for the moment it contains only your command, perhaps in the future we will have other commands related to the java project)